README.md

```diff
@@ -840,7 +840,7 @@ Run the stress-ng workload with the given parameters
         </details><details><summary>verbose (<code>bool</code>)</summary>
         <table><tbody><tr><th>Name:</th><td>Verbose</td></tr><tr><th>Description:</th><td width="500">Verbose output</td></tr><tr><th>Required:</th><td>No</td></tr><tr><th>Type:</th><td><code>bool</code></td></tbody></table>
         </details><details><summary>workdir (<code>string</code>)</summary>
-        <table><tbody><tr><th>Name:</th><td>Working Directory</td></tr><tr><th>Description:</th><td width="500">Directory in which stress-ng will be executed (for example, to target a specific volume)</td></tr><tr><th>Required:</th><td>No</td></tr><tr><th>Default (JSON encoded):</th><td><pre><code>&#34;.&#34;</code></pre></td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
+        <table><tbody><tr><th>Name:</th><td>Working Directory</td></tr><tr><th>Description:</th><td width="500">Directory in which stress-ng will be executed (for example, to target a specific volume)</td></tr><tr><th>Required:</th><td>No</td></tr><tr><th>Default (JSON encoded):</th><td><pre><code>&#34;/tmp&#34;</code></pre></td></tr><tr><th>Type:</th><td><code>string</code></td></tbody></table>
```

Comment:
This is fascinating and disturbing:  the docsgen bot seems to have decided that, when the _user_ executes `tempfile.gettempdir()` it will produce the value `/tmp`...which is quite possibly true, but [it can _also_ produce](https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir) the value of the `TMPDIR`, `TEMP`, or `TMP` environment variables, `/tmp`, `/var/tmp`, or `/usr/tmp` (whichever one it finds first to be writable), or, as a last resort, the current working directory (which, presumably, is `'.'`).  🙃

I'm not quite sure what we could or should have done to address this, but it's kind of a bummer that the bot is too clever by half, here.  😐

Reply:
I never looked at the bot code -- but it can't be just parsing the Python source, and the most likely alternative is starting the plugin and querying the "live" schema. Therefore, it would know nothing about the documented semantics of `tempfile.gettempdir` and only knows or cares about the actual default value in the schema -- which is just the returned file path, regardless of how it was derived.

So I don't think it's "too clever" -- I suspect it's simplistically pragmatic, and that if the schema were built in an environment where `TMPDIR=/home/seagull/mine`, it'd just as confidently (and "correctly" if misleadingly) tell you that the default was `/home/seagull/mine`.

Reply:
I concur.  And, the problem is that the documentation is recording and reporting a value as "fixed" when it's actually sensitive to the _user's_ environment.

But, hopefully, it won't actually matter to anyone.  (Probably 95% of users won't note or depend on this bit of the documentation; 90% of those who do will expect and receive `/tmp`; 50% of the remainder will likely be "delighted" to find that the value follows their `TMPDIR` definition...hopefully, for the remaining two out of a thousand users, the pain won't be too great, and it's easy enough for them to address by overriding the default.)